### PR TITLE
Rg 9

### PR DESCRIPTION
**Pull request by AntonFartovii**
Created sticky header and footer. Replace buttons singin, signout, logout in header.

Check `npm run build` and `npm run start`
image
image

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -18,27 +18,7 @@ const Welcome = () => {
 
   return (
     <>
-      <LangSwitcher />
       <Sidebar />
```
Comment (sekaa4) [inline src/pages/index.tsx:21]: you dont need this component on the welcome page

File: src/pages/main/index.tsx [hunk 2]
```diff
@@ -36,11 +36,7 @@ const Home = (props: SSRPageProps) => {
 
   return (
     <>
-      <LangSwitcher />
       <Sidebar />
-      <Button variant="contained" onClick={() => logout()}>
-        LogOut
-      </Button>
       <input className={homeStyles.input} />
       <main className={homeStyles.editor}>
```
Comment (sekaa4) [inline src/pages/main/index.tsx:41]: you have main tag in layout, why do you use this tag here one more time

Comment (AntonFartovii) [reply src/pages/main/index.tsx:41]: I will fix this issue in RG-10

File: src/pages/index.tsx (hunk 1 above)
Comment (AntonFartovii) [reply src/pages/index.tsx:21]: I will fix this issue in RG-10

**Review by sekaa4 — APPROVED** [review]

**Review by ImmelstronDev — APPROVED** [review]

**Pull request merged**